shared/java/javastyle.gradle

```diff
@@ -48,7 +48,7 @@ spotless {
         target fileTree('.') {
             include '**/*.json'
             exclude '**/build/**', '**/build-*/**', '**/bin/**'
-            exclude '**/simgui-ds.json', '**/simgui-window.json', '**/simgui.json', '**/networktables.json', '**/*test.json'
+            exclude '**/simgui-ds.json', '**/simgui-window.json', '**/simgui.json', '**/networktables.json', '**/*test.json', '**/camera calibration.json'
```

Comment:
Is this file created by wpical?

Reply:
Yes, during the tests, it runs the camera calibration which saves to a .json file.

Reply:
Is there a reason we can't use a temporary file instead of polluting the source tree?